sdk/healthinsights/Azure.Health.Insights.CancerProfiling/cadl-location.yaml

```diff
@@ -0,0 +1,6 @@
+directory: specification/cognitiveservices/HealthInsights/healthinsights.oncophenotype/
+additionalDirectories:
+  - specification/cognitiveservices/HealthInsights/healthinsights.autorest/
+  - specification/cognitiveservices/HealthInsights/healthinsights.common/
+commit: 7ecc7d7d636976bbe19020a1edd01ce288df734b
+repo: asaflevi-ms/azure-rest-api-specs
```

Comment:
```suggestion
should use the public repo 
```

Reply:
I cannot use public repo as it is not merged yet

Reply:
Then we may delay to merge this PR.

Reply:
After converting to typespec.
I will update the repo to main, once it will be merged.

Reply:
tsp-location updated to reference public repo.